api/turing/it/database/database.go

```diff
@@ -9,15 +9,14 @@ import (
 	"testing"
 	"time"
 
+	"github.com/gojek/turing/api/turing/config"
+	"github.com/gojek/turing/api/turing/db"
 	"github.com/gojek/turing/api/turing/log"
-	gomigrate "github.com/golang-migrate/migrate/v4"
-	"github.com/golang-migrate/migrate/v4/database/postgres"
-	_ "github.com/golang-migrate/migrate/v4/source/file"
 	"github.com/jinzhu/gorm"
 )
 
 func connectionString(db string) string {
```

Comment:
This can be moved to `db`package as well (as it's used there as well), with a modification:
```
func ConnectionString (cfg *config.DatabaseConfig) string {...}
```

Reply:
done